tests/test_quaternion.py

```diff
@@ -147,7 +147,7 @@ def test_from_ra_dec_roll(self, arr):
             [s1 * s3 - c1 * c3 * s2, c3 * s1 + c1 * s2 * s3,  c1 * c2]   # noqa
         ]))
 
-        assert Quaternion.from_ra_dec_roll(*arr) == Quaternion.from_matrix(expected)
```

Comment:
I'd like to explicitly add the `@example` you wrote in the issue, as this will clarify why we need this tolerance.

Reply:
done